util/grpc/grpc.go

```diff
@@ -87,7 +88,7 @@ func BlockingDial(ctx context.Context, network, address string, creds credential
 			grpc.FailOnNonTempDialError(true),
 			grpc.WithContextDialer(dialer),
 			grpc.WithTransportCredentials(insecure.NewCredentials()), // we are handling TLS, so tell grpc not to
-			grpc.WithKeepaliveParams(keepalive.ClientParameters{Time: 10 * time.Second}),
+			grpc.WithKeepaliveParams(keepalive.ClientParameters{Time: 20 * time.Second}),
```

Comment:
Question because I'm not too bright: why is this 2x the keepalive enforcement policy minimum? 

If the minimum ends up in commons.go, should we just use `2 * commons.GRPCKeepAliveEnforcementMinimum` here?

Reply:
Makes sense! Want to add something about that as a comment? Just so future readers understand more easily.